lib/system.nim

```diff
@@ -3289,6 +3289,11 @@ proc `/=`*[T: float|float32](x: var T, y: T) {.inline, noSideEffect.} =
 
 proc `&=`* (x: var string, y: string) {.magic: "AppendStrStr", noSideEffect.}
 
+proc `&=`*[T](a: var seq[T], b: seq[T]) =
```

Comment:
Er, no,  the signature needs to be: 

proc `&=`*[T](a: var seq[T]; b: T) = ...